apps/api/src/app/environments-v1/usecases/output-renderers/in-app-output-renderer.usecase.ts

```diff
@@ -61,26 +61,26 @@ const InAppRenderOutputSchema = z.object({
   avatar: z.string().optional(),
   primaryAction: z
     .object({
-      label: z.string(),
+      label: z.string().optional(),
```

Comment:
I don't think this is the right approach, it's invalid to have a primary action without a label, etc. The correct fix IMO is to not send an empty object for `primaryAction` when the primary action isn't needed.

Reply:
Are you talking about not sending an empty object from the FE to the API?